src/repositories/causeRepository.test.ts

```
@@ -0,0 +1,412 @@
+
+      // Try to validate the same title with extra whitespace
+      try {
+        await validateCauseTitle('  Test Cause Title  ');
+        assert.fail('Should have thrown an error');
+      } catch (e) {
+        assert.equal(e.message, 'Cause title already exists');
+      }
+    });
+  });
+
+  describe.only('findAllCauses test cases', () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)